www/front_src/src/Resources/translatedLabels.ts

```diff
@@ -190,6 +190,7 @@ export const labelAddComment = 'Add a comment';
 export const labelCommentAdded = 'Comment added';
 export const labelDisplayEvents = 'Display events';
 export const labelExportToPng = 'Export to png';
+export const labelGoToPerformancePage = 'Performance page';
```

Comment:
Why not `Go to performance page` ?

Reply:
I don't know, you think is better to write this?

Reply:
I mean, your label is labelGoToPerformancePage, that's why it confused me

Reply:
I push a new commit to change the name of variable ;)